pkgs/development/libraries/drogon/no_unused_parameter.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f4b3b25..200870e 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -71,6 +71,7 @@ endif (BUILD_DROGON_SHARED)
+ 
+ if (CMAKE_CXX_COMPILER_ID MATCHES GNU)
+     target_compile_options(${PROJECT_NAME} PRIVATE -Wall -Wextra -Werror)
++    set(CMAKE_CXX_FLAGS -Wno-unused-parameter)
```

Comment:
This isn't nix-specific, should probably be upstreamed

Reply:
I did some more experimenting and I discovered that this patch is not needed if you include `openssl`.  For now I'll just include it, but I'll see if I can get this fixed upstream so I can make including `openssl` override-able.